Cargo.toml

```diff
@@ -35,7 +35,7 @@ bitflags = "1.2.1"
 log = { version = "0.4.11", default-features = false }
 ucs2 = "0.3.1"
 uefi-macros = "0.3.2"
-
+arrayvec = { version = "0.7.1", default-features = false }
 [workspace]
```

Comment:
Please keep a blank line between the `dependencies` section and the `workspace` section.